tsdb/chunks/head_chunks.go

```diff
@@ -376,7 +376,29 @@ func repairLastChunkFile(files map[int]string) (_ map[int]string, returnErr erro
 	if err != nil {
 		return files, errors.Wrap(err, "file stat during last head chunk file repair")
 	}
-	if info.Size() == 0 {
+
+	remove := info.Size() == 0 // Empty file.
+	if info.Size() >= MagicChunksSize {
```

Comment:
nit: I think it might be better to move this if block to a helper function so the code can read like: 

```
remove := info.Size() ==0 
remove := isFileCorrupted(...)

if remove {
 ....
}
```

Reply:
Since this logic is only used here and that `repairLastChunkFile` is not too huge, I prefer to not create another function here.

Reply:
Make sense.